apps/core/components/sectorDetail/segment/Segment.js

```diff
@@ -0,0 +1,88 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { StyleSheet } from '@kiwicom/universal-components';
+import {
+  TimelineTransportDetail,
+  TimelineDate,
+  Text,
+} from '@kiwicom/margarita-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { graphql, createFragmentContainer } from '@kiwicom/margarita-relay';
+
+import SegmentStopInfo from './SegmentStopInfo';
+import { mockTransporterData } from './SegmentConstants';
+import {
+  getFormattedDate,
+  dateFormat,
+} from '../../ItineraryCard/TripSectorHelpers';
+import type { Segment as SegmentType } from './__generated__/Segment.graphql';
+
+type Props = {|
+  +data: ?SegmentType,
+|};
+
+class Segment extends React.Component<Props> {
+  transporterInfoHeader = (expanded: boolean) => (
+    <View>
+      <Text>Transport info {expanded ? 'expanded' : 'closed'}</Text>
+    </View>
+  );
+
+  render() {
+    return (
+      <View>
+        <TimelineDate
+          formattedDate={getFormattedDate(
+            this.props.data?.departure?.time?.local,
+            dateFormat,
+          )}
+          containerStyle={styles.timelineDate}
+        />
+        <SegmentStopInfo
+          data={this.props.data?.departure}
+          typeLabel="Departure"
+        />
+        <TimelineTransportDetail
+          carrier={{
+            code: this.props.data?.transporter?.name ?? '',
+            name: 'Carrier name', // @TODO - mock value, value is not returned from api yet
+            type: 'airline', // @TODO - mock value, carrier_type value is not returned from api yet
```

Comment:
May I ask you for creating the issue for that? thanks

Reply:
Definitely, issue created: https://github.com/kiwicom/margarita/issues/435

Reply:
thanks